### PR TITLE
docs: translate `faq-styling`

### PR DESCRIPTION
**Pull request by huantingwei**
Hi, sorry for the mistake in the last PR.
Please feel free to make any edit! Thanks!

### CODE REVIEW

File: content/docs/faq-styling.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ layout: docs
 category: FAQ
 ---
 
-### How do I add CSS classes to components? {#how-do-i-add-css-classes-to-components}
+### 如何在 components 中加入 CSS classes？ {#how-do-i-add-css-classes-to-components}
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:9]: 標題內的英文開頭我們都以大寫為主，然後為單數：

```suggestion
### 如何在 Component 中加入 CSS Class？ {#how-do-i-add-css-classes-to-components}
```

File: content/docs/faq-styling.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ layout: docs
...
 
-Pass a string as the `className` prop:
+把字串以 prop `className` 傳入：
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:11]:
```suggestion
傳遞一個 string 作為 `className` 的 prop：
```

File: content/docs/faq-styling.md [hunk 3]
```diff
@@ -6,17 +6,17 @@ layout: docs
...
 
 ```jsx
 render() {
   return <span className="menu navigation-menu">Menu</span>
 }
 ```
 
-It is common for CSS classes to depend on the component props or state:
+CSS class 常會依賴 component prop 或 state：
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:19]:
```suggestion
通常 CSS class 常會依賴 component 的 prop 或 state：
```

File: content/docs/faq-styling.md [hunk 4]
```diff
@@ -28,24 +28,24 @@ render() {
 }
 ```
 
->Tip
->
->If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
+>小技巧
+>
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:32]: Unnecessary empty

File: content/docs/faq-styling.md [hunk 5]
```diff
@@ -28,24 +28,24 @@ render() {
...
 
->Tip
->
->If you often find yourself writing code like this, [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) package can simplify it.
+>小技巧
+>  
+>如果你發現你常寫類似的程式碼，可使用 [classnames](https://www.npmjs.com/package/classnames#usage-with-reactjs) 函式庫簡化。
 
-### Can I use inline styles? {#can-i-use-inline-styles}
+### 可以使用 inline style 嗎？ {#can-i-use-inline-styles}
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:35]:
```suggestion
### 可以使用 Inline Style 嗎？ {#can-i-use-inline-styles}
```

File: content/docs/faq-styling.md [hunk 6]
```diff
@@ -28,24 +28,24 @@ render() {
...
 
-Yes, see the docs on styling [here](/docs/dom-elements.html#style).
+可以，請見 [此處](/docs/dom-elements.html#style) 文件關於 styling  部分。
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:37]:
```suggestion
可以，請見[此處](/docs/dom-elements.html#style) 文件關於 styling 部分。
```

File: content/docs/faq-styling.md [hunk 7]
```diff
@@ -28,24 +28,24 @@ render() {
...
 
-### Are inline styles bad? {#are-inline-styles-bad}
+### inline styles 不好嗎？ {#are-inline-styles-bad}
```
Comment (neighborhood999) [inline content/docs/faq-styling.md:39]:
```suggestion
### Inline Style 不好嗎？ {#are-inline-styles-bad}
```

**Review by neighborhood999 — COMMENTED** [review]
Thanks for helping translate! Just some minor suggestions and changes.

File: content/docs/faq-styling.md (hunk 3 above)
Comment (huantingwei) [reply content/docs/faq-styling.md:19]: How about 
`通常 CSS class 會依賴 component 的 prop 或 state：`

Comment (neighborhood999) [reply content/docs/faq-styling.md:19]: It's good!

File: content/docs/faq-styling.md (hunk 4 above)
Comment (huantingwei) [reply content/docs/faq-styling.md:32]: I think empty on line 32 was on the original file too.

**Comment by neighborhood999** [comment]
Good! Thanks for helping.

**Pull request merged**